<launch>

  <node name="rostopic_publisher" pkg="rostopic" type="rostopic" args="pub topic_name std_msgs/String 'Started' -r 1" clear_params="True">
    <param name="capability_group" value="autostart" />
    <param name="default_cfg/autostart/delay" value="5.0" />
  </node>

  <node name="rostopic_echo" pkg="rostopic" type="rostopic" args="echo topic_name" clear_params="True">
    <param name="capability_group" value="autostart" />
    <param name="default_cfg/autostart/exclude" value="false" />
    <param name="default_cfg/autostart/delay" value="5.0" />
    <param name="default_cfg/autostart/required/publisher" value="topic_name" />
  </node>

  <node respawn="True" name="rostopic_respawn" pkg="rostopic" type="rostopic" args="what" clear_params="True">
    <param name="capability_group" value="autostart" />
    <param name="respawn/max" value="5" />
    <param name="respawn/min_runtime" value="60" />
    <param name="respawn/delay" value="2" />
  </node>


</launch>
